### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by 3clypse**
# Purpose
Add dependabot dependency check in order to maintain Go and Docker dependencies up-to-date and security updates on time.

https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#enabling-or-disabling-dependabot-security-updates-for-an-individual-repository

# Why
- A way to keep dependencies updates in single PR's that can be visible to anyone.
- Security patches could be applied quicker, given the bot will check for updates every day.

## + Info
- https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
- https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  # Enable version updates for go modules
+  - package-ecosystem: "gomod"
+    # Look for Go modules in the `root` directory
```
Comment (gotjosh) [inline .github/dependabot.yml:5]:
```suggestion
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    directory: "/"
+    # Check for updates every day (weekdays)
+    schedule:
+      interval: "daily"
```
Comment (gotjosh) [inline .github/dependabot.yml:9]:
```suggestion
      interval: "weekly"
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  # Enable version updates for Docker
+  - package-ecosystem: "docker"
+    # Look for a `Dockerfile` in the `root` directory
+    directory: "/"
+    # Check for updates once a week
+    schedule:
+      interval: "weekly"
```
Comment (gotjosh) [inline .github/dependabot.yml:17]:
```suggestion
  - package-ecosystem: "docker"
    directory: "/"
    schedule:
      interval: "weekly"
```

**Review by gotjosh — COMMENTED** [review]
Can you please remove the comments and set the interval of gomod to weekly?

**Comment by 3clypse** [comment]
> Can you please remove the comments and set the interval of gomod to weekly?

Done! Thank you for looking at this :)

**Review by gotjosh — APPROVED** [review]
LGTM.

@roidelapluie @simonpasquier what do you think?

**Comment by gotjosh** [comment]
Can you try re-pushing to your branch again to see if the CI triggers? I can't seem to find the CI run of this PR on CircleCI to try and let it run.

**Comment by 3clypse** [comment]
> Can you try re-pushing to your branch again to see if the CI triggers? I can't seem to find the CI run of this PR on CircleCI to try and let it run.

I bring the latest changes from prometheus/alertmanager <main> and re-commited my suggestion change.
Let me know if I can do something else.

Thank you.

**Comment by gotjosh** [comment]
Thanks!

**Pull request merged**